Answered some of the questions on website

Bruhris opened the pull request.

devvsakib reviewed (changes requested):
i hope you didnt read README.md file. please read this file for proper instruction. add-questions.html file only for adding question. there are questions-answers.html, where you can answers or add questions with answers. do not delete any question

Bruhris commented:
Reverted the changes I made to the add-questions.html page and added answers to questions-answers.html page. Let me know if there's anything wrong with them.
